dist/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+  <title>My Weather App</title>
+    <script defer src="main.js"></script>
+</head>
+<body>
+    <section class="container">
+        <div class="weather-container">
+            <h1 class="title" id="city"></h1>
+            <span id="time"></span>
+            <h3 id="temp"></h3>
+            <i class="fal fa-acorn"></i>
+            <span id="description"></span>
+            <span id="humidity"></span> 
+            <p id="visibility"></p> 
+            <span id="wind"></span>       
+        </div>
+        <div class="search-container">
+            <h1 class="title">Current Weather</h1>
+            <h4 id="search">Search Location</h4>
+            <i class="fas fa-thumbtack"></i>
+            <form action="" id="form">
+                <input type="text" placeholder="Enter city" id="location">
+                <button type="submit" id="submitBtn">submit</button>
+            </form>
+            <div>
+                <label class="switch" for="switch">
+                    <input type="checkbox">
+                    <span class="slider round"></span>
```

Comment:
[Required]
- [ ] Make sure the slider button is working and change the temperature to Celsius or Fahrenheit.